static/styles/popovers.scss

```diff
@@ -194,6 +194,10 @@ ul.remind_me_popover .remind_icon {
     font-size: 15px;
 }
 
+.user-profile-modal-field-value[data-type="Long text"] {
+    white-space: pre;
```

Comment:
Shouldn't this be `pre-wrap`, so that if they have a really long line in their text, it doesn't get messed up?

(As a sidenote, I guess we will be able to remove this logic if/when we move this to markdown, since that would handle the newlines for us; @shubhamdhama FYI)

Reply:
Is this the preferred behavior? @timabbott 

![screenshot at aug 08 11-00-55](https://user-images.githubusercontent.com/15116870/43855403-57c2c2da-9afa-11e8-8298-350673089aab.png)
![screenshot at aug 08 11-00-46](https://user-images.githubusercontent.com/15116870/43855404-57e03306-9afa-11e8-8812-1ce8379a21b9.png)

Reply:
The preferred behavior is the same behavior we have in the main Zulip message feed for normal content: (1) we break at word boundaries where possible, and (2) we don't overflow to scrolling (code blocks are a special case).

Reply:
@timabbott fixed!
![screenshot at aug 08 15-14-51](https://user-images.githubusercontent.com/15116870/43867370-d8baec96-9b1d-11e8-9ae6-32dbf96a4973.png)